website/blog/2022-07-27-getting-started-with-the-dbt-semantic-layer.md

```diff
@@ -0,0 +1,165 @@
+---
+title: "Getting Started With The dbt Semantic Layer"
+description: "There's been a lot of discussion around dbt Metrics and what they are. Callum Mccann clarifies what the Semantic Layer is, what is available today, and how one can already begin to use parts of it!"
+slug: getting-started-with-the-dbt-semantic-layer
+
+authors: [callum_mccann]
+
+tags: [dbt product updates]
+hide_table_of_contents: false
+
+date: 2022-07-27
+is_featured: true
+---
+
+# Getting Started With The dbt Semantic Layer
+
+> TLDR: The Semantic Layer is made up of a combination of open-source and SaaS offerings and is going to change how you consume and define metrics.
+
+
+In recent months, we’ve seen a lot of excitement (and some understandable confusion) in the community around what the dbt Semantic Layer is. This is completely understandable, as elements of this “layer” are currently available while others aren’t. To those who aren’t following this saga with the intensity of [someone watching their investments on the crypto market](https://mobile.twitter.com/scannergr1/status/1536198701215109122/photo/1), we hope this blog can provide clarification on the following things:
+
+1. What is the dbt semantic layer?
+2. How do I use it?
+3. What is publicly available now?
+4. What is still in development?
+
+With that, lets get into it!
+
+<!--truncate-->
+
+> Some of you might be confused - you’ve seen us talk about the Metrics Layer in the past and now we’re talking about the Semantic Layer. Well, what was previously called the Metrics Layer has been re-branded to the “Semantic Layer” in order to better reflect our long term plans. Today’s functionality focus largely on metrics but tomorrow’s will expand to be a programmatic interface to the DAG, to better understand entities and what models represent, and much more. Drew’s post about the future of the Semantic Layer explains this far better.
```

Comment:
Is rebranded the right word here? Maybe we say 'As we evaluated the long term plans for this part of our tech stack, we came to see that we could more accurately reflect it's capabilities as with the name "Semantic Layer" as opposed to "Metrics Layer". For a more in depth explanation of this see Drew's post.'

Reply:
Updated!